StatBot/Commands/TuneStat.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Bot.Schema;
+using StatBot.Interfaces;
+using StatBot.Models;
+using System.Collections.Generic;
+using StatBot.Cards;
+using System;
+
+namespace StatBot.Commands
+{
+    [Serializable]
+    public class TuneStat : ITool
+    {
+        public string Description { get; set; }
+        public List<string> CommandsName { get; set; }
+        public bool IsAdmin { get; set; }
+
+        public Activity Run(Activity activity)
+        {
+            if (activity?.Conversation != null)
+            {
+                activity.Attachments = new List<Attachment>();
+
+                Attachment attachment;
+                var actions = new List<CardAction>();
+
+                foreach (var stat in DataModel.Statistics)
+                {
+                    if (stat.IsActive)
+                        actions.Add(new StatisticButton(stat.Id, stat.Name,"Subs").Action);
+                }
+
+                Attachment a = new Attachment()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'a' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1481)